docs/installation.md

```diff
@@ -1,9 +1,11 @@
-# Clone the repository
+# Installation and Configuration for cfgov-refresh
+
+## Clone the repository
 
 Using the console, navigate to the root directory in which your projects live and clone this project's repository:
 
 ```bash
-git clone git@github.com:cfpb/cfgov-refresh.git
+git clone https://github.com/cfpb/cfgov-refresh.git
```

Comment:
Why http instead of git as the protocol? I find whenever I clone remotes via https I have to enter my username/password at the command line when pushing, and it doesn't work on our work machines.